docs/manual/map-key-checker.tex

```diff
@@ -51,19 +51,70 @@ \section{Map key annotations\label{map-key-annotations}}
 
 \section{Default annotations\label{map-key-defaults}}
 
-The null literal is implicitly annotated with \code{@UnknownKeyFor}, because
-\code{null} might be a key for any map.
+The \code{null} literal is implicitly annotated with \code{@UnknownKeyFor}
+If \code{null} were \code{@UnknownKeyFor}, that would mean that
```

Comment:
UnknownKeyFor should be KeyForBottom here, right?

Reply:
You are right.  Thanks for the catch!